cloudbuild.yaml.in

```diff
@@ -0,0 +1,24 @@
+# Cloud Builder pipeline
+# https://cloud.google.com/container-builder/docs/overview
+
+steps:
+# Perform maven build, omitting local docker operations
+- name: 'maven:3.3.9-jdk-8'
+  args: ['mvn', '-P-local-docker-build', '-P-test.local', 'clean', 'install']
+# Build the runtime container
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=${IMAGE}', '--no-cache', 'jetty9/target/docker']
+
+# Test the built image
+
+# Run our own integration tests
+# TODO this requires the cloud sdk to be installed
```

Comment:
The "remote" integration tests do:
https://github.com/GoogleCloudPlatform/jetty-runtime/tree/master/tests

(the ones that are enabled by running with -Ptest.remote)